src/stories/README/Box.md

```diff
@@ -0,0 +1,14 @@
+# Box
+
+Displays a block level element.
+
+The basic building block of a rendition application.
+
+[View story source](https://github.com/resin-io-modules/rendition/blob/master/src/stories/Box.js)
+
+## Props
+
+| Name          | Type      | Default   | Required   | Description                                          |
+| --------------------------------------------------------------------------------------------------------- |
+| `flex`      | <code>string &#124; string[]</code> | - | - | Sets `flex`, if the value is an array, sets a responsive style corresponding to the theme's breakpoints
```

Comment:
I think that we also have `order` in the typings.
It feels very poor that we do not mention at all any of the DefaultProps :thinking: even though the story does demonstrate some of them.

Reply:
@thgreasi In the main README we mention that **all** components utilise the styled system props. I'm open to suggestions on how we can make that clearer